qiskit_experiments/library/tomography/tomography_analysis.py

```diff
@@ -91,6 +92,12 @@ def _default_options(cls) -> Options:
                 tomographic preparations.
             target (Any): Optional, target object for fidelity comparison of the fit
                 (Default: None).
+            target_bootstrap_samples (int): Optional, number of boostrapped samples to run
```

Comment:
Bootstrapped samples sounds bit unclear to users without reading the code. Clarifying resampling outcomes based on the observed probability by assuming multinominal distribution seems better to me.